frontend/test/cypress/integration/product/pro_user.cy.ts

```diff
@@ -0,0 +1,31 @@
+describe('Pro user test', () => {
+   it('will give pro users a discount', () => {
+      cy.loadProductPageByPath('/products/repair-business-toolkit');
+
+      // Get price from page
+      cy.findAllByTestId('product-price')
+         .first()
```

Comment:
Is there a possibility of the price for a recommended product being added to the list first? In the case, the actual product price does not render or is slower at rendering?